### PR TITLE
[Testing] chore: add make targets & update feature path logging

### PR DESCRIPTION
**Pull request by bryanchriswhite**
## Summary

- Factor new make target, `test_e2e_env`, out of `test_e2e`.
- Add make target shortcuts for specific E2E features:
  - `test_e2e_claim_proof` to run session.feature
  - `test_e2e_settlement` to run tokenomics.feature
  - `test_load_relays_stress` to run the relay stress load test

## Issue

- [MVT features table](https://www.notion.so/buildwithgrove/Minimum-Viable-Testnet-Success-Criteria-baa2392d0c934f408a277c6afa1b226c?pvs=4#e9bed3470316427998c8ab604bbfdff6)

## Type of change

Select one or more:

- [ ] New feature, functionality or library
- [ ] Bug fix
- [x] Code health or cleanup
- [ ] Documentation
- [ ] Other (specify)

## Testing

- [ ] **Unit Tests**: `make go_develop_and_test`
- [ ] **LocalNet E2E Tests**: `make test_e2e`
- [ ] **DevNet E2E Tests**: Add the `devnet-test-e2e` label to the PR. **THIS IS VERY EXPENSIVE**, so only do it after all the reviews are complete.
- [ ] **Documentation changes**: `make docusaurus_start`

## Sanity Checklist

- [ ] I have tested my changes using the available tooling
- [ ] I have commented my code
- [ ] I have performed a self-review of my own code; both comments & source code
- [ ] I create and referenced any new tickets, if applicable
- [ ] I have left TODOs throughout the codebase, if applicable

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -271,13 +271,24 @@ go_imports: check_go_version ## Run goimports on all go files
...
 #############
 
-.PHONY: test_e2e
-test_e2e: acc_initialize_pubkeys_warn_message ## Run all E2E tests
+.PHONY: acc_initialize_pubkeys_warn_message test_e2e_env
+test_e2e_env: ## Setup the default env vars for E2E tests
 	export POCKET_NODE=$(POCKET_NODE) && \
 	export APPGATE_SERVER=$(APPGATE_SERVER) && \
-	POKTROLLD_HOME=../../$(POKTROLLD_HOME) && \
+	export POKTROLLD_HOME=../../$(POKTROLLD_HOME)
```
Comment (Olshansk) [inline Makefile:278]:
```suggestion
	export POKTROLLD_HOME=../../$(POKTROLLD_HOME) # because E2E tests live under e2e/tests
```

File: Makefile [hunk 2]
```diff
@@ -271,13 +271,24 @@ go_imports: check_go_version ## Run goimports on all go files
...
+
+.PHONY: test_e2e
+test_e2e: test_e2e_env ## Run all E2E tests
 	go test -v ./e2e/tests/... -tags=e2e,test
 
+.PHONY: test_e2e_claim_proof
+test_e2e_claim_proof: test_e2e_env ## Run only the E2E suite that exercises the claim/proof life-cycle
```
Comment (Olshansk) [inline Makefile:285]:
```suggestion
.PHONY: test_e2e_session
test_e2e_session: test_e2e_env ## Run only the E2E suite that exercises the session (i.e. claim/proof) life-cycle
```

File: Makefile [hunk 3]
```diff
@@ -271,13 +271,24 @@ go_imports: check_go_version ## Run goimports on all go files
 ### Tests ###
 #############
 
-.PHONY: test_e2e
-test_e2e: acc_initialize_pubkeys_warn_message ## Run all E2E tests
+.PHONY: acc_initialize_pubkeys_warn_message test_e2e_env
+test_e2e_env: ## Setup the default env vars for E2E tests
```
[the remark this answers is not in the document]
Comment (bryanchriswhite) [reply Makefile:275]: Nice catch - thanks! :sweat_smile:

File: Makefile (hunk 1 above)
Comment (Olshansk) [reply Makefile:278]: After thinking about it, this is extra unnecessary information. Resolving and ignoring.

File: Makefile [hunk 4]
```diff
@@ -282,13 +282,28 @@ go_imports: check_go_version ## Run goimports on all go files
...
 
+.PHONY: test_e2e_session
+test_e2e_session: test_e2e_env ## Run only the E2E suite that exercises the session (i.e. claim/proof) life-cycle
+	go test -v ./e2e/tests/... -tags=e2e,test --features-path=session.feature
+
+.PHONY: test_e2e_settlement
+test_e2e_settlement: test_e2e_env ## Run only the E2E suite that exercises the session & tokenomics settlement
+	go test -v ./e2e/tests/... -tags=e2e,test --features-path=0_settlement.feature
+
+.PHONY: test_load_relays_stress
```
Comment (Olshansk) [inline Makefile:303]: I don't think this exists yet. Assuming it'll come soon though?

Not going to block.

Comment (bryanchriswhite) [reply Makefile:303]: Yes, it's in #432.

**Pull request merged**